tests/ctags/enum.f90.tags

```diff
@@ -11,6 +9,8 @@ Named5
 Named6�2�Constants�0
 Named7�2�Constants�0
 a�4�Enum#1�0
+anon_enum_1�2�Constants�1
+anon_enum_2�2�Constants�1
```

Comment:
I wonder if the new naming is really appreciated by Fortran users? I guess we'll see ;-)

Reply:
Yeah. But the new naming is now universal for all languages and we won't need to handle each of them separately. Hopefully none of the 2 users who use Geany for Fortran coding will complain ;-).

Reply:
Yea, lets see if there's any (negative) feedback.